apps/books/templates/categoryvisebooks.html

```diff
@@ -0,0 +1,28 @@
+<h1>{{ object.title }}:</h1>
+<ul style="list-style-type: none;">
+    {% for book in object.book_set.all %}
+    <li>
+        <div style="background-color: rgb(240, 240, 240); width: 20%; height: 300px; float: left; padding: 20px; margin: 17px;">
+            <div style="width: 100%; height:240px;">
+                <a href="#">
+                    <img src="{{ book.cover_image.url }}" alt="{{ book.title }}" style="width: 100%; height: 100%;">
+                </a>
+            </div>
+            <div style="text-align: center; width:100%; height: 50px;">
+                <p style="margin: 3px 0px;">{{ book.title }} By: {{ book.author }}</p>
+                {% if book.discount_price %}
+                    <p style="margin: 3px 0px;">
+                        <del>Rs: {{ book.price }}</del>
+                        Rs: {{ book.discount_price }}
+                    </p>
+                {% else %}
+                    <p style="margin: 3px 0px;">Rs: {{ book.discount_price }}</p>
+                {% endif %}
+                <a href="#">
+                    <button style="padding: 3px;">Add to Cart</button>
+                </a>
+            </div>
+        </div>
```

Comment:
How about if make a separate html `book.html` and then we can reuse it on multiple pages ? like on homepage and on category wise books page.. WDYT?